requirements.txt

```diff
@@ -0,0 +1,7 @@
+f90nml>=1.1.0
+appdirs>=1.4.0
+requests>=2.22.0
+pyyaml>=5.0
+gcsfs>=0.4.0
+fsspec>=0.6.0
```

Comment:
Kind of unrelated, but can you increase these versions to I think 0.6.0 and 0.8.0? Whatever it is in fv3net. The interim versions have caused incompatibility issues when gcsfs is newer and fsspec is older.